inst/templates/BS4/content-article-index.html

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="col-md-9 contents">
-    {{#logo}}<img src="{{logo.src}}" id="{{logo.id}}" alt="" >{{/logo}}
-    <div class='pb-2 mt-4 mb-2 border-bottom'>
+    {{#logo}}<img src="{{logo.src}}" class="pkg-logo" alt="" >{{/logo}}
```

Comment:
Instead of these conditional classes how about just making `.page-header` have a minimum height?

Reply:
That's a great idea, I'll try that.